src/backend/api/meals.js

```diff
@@ -2,11 +2,111 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+//Returns all meals
 router.get("/", async (request, response) => {
+  let maxPrice = 100000;
```

Comment:
This is excellent! I've seen so many PRs that try to do this with a bunch of if statements instead of starting with some default values! 👏🏼